README.md

```diff
@@ -31,25 +31,20 @@ npm install elasticsearch
 
 ## Use in the Browser
 
-Check out the [Browser Builds](https://www.elastic.co/guide/en/elasticsearch/client/javascript-api/16.x/browser-builds.html) doc page for help downloading and setting up the client for use in the browser.
+Check out the [Browser Builds](https://www.elastic.co/guide/en/elasticsearch/client/elasticsearch-js/16.x/browser-builds.html) doc page for help downloading and setting up the client for use in the browser.
 
 ## Docs
```

Comment:
Suggestion, let's remove all the links and add the message:
_Documentation can be found on elastic.co_ with the link to https://github.com/elastic/docs/pull/2318